task.sql

```diff
@@ -2,10 +2,19 @@
 USE ShopDB; 
 
 -- Some data should be created outside the transaction (here)
+INSERT INTO Orders (CustomerID, Date)
+VALUES (1, '2024-02-06');
 
 -- Start the transaction 
 START TRANSACTION; 
 
 -- And some data should be created inside the transaction 
+INSERT INTO OrderItems (OrderID, ProductID, Count)
+VALUES (1, 1, 1);
 
-COMMIT; 
\ No newline at end of file
+UPDATE Products
+SET WarehouseAmount = WarehouseAmount - 1
+WHERE Products.ID = 1;
+
+COMMIT;
+ 
```

Comment:
You add space in last line, so in doesn't work as EOF
![image](https://github.com/mate-academy/rda_task_3_transactions/assets/87720583/9cbf1868-e3d7-429f-96cf-9fc5e1a0a8ad)